egui/src/lib.rs

```diff
@@ -227,6 +227,7 @@
 //! ui.style_mut().wrap = Some(false);
 //! ```
 
+#![cfg_attr(not(debug_assertions), deny(rustdoc::missing_doc_code_examples))] 
```

Comment:
Shouldn't this be `rustdoc::missing_crate_level_docs` ?

Reply:
Yes.  Definitely. Copy and paste error on my part.  Sorry.  Fixing....

Reply:
I am having trouble getting any version of `missing_crate_level_docs` to work though. That is, if I remove the crate level docs I don't get any warnings.

Reply:
I am seeing the same behavior, no warnings if I remove the crate level comments.

Looks like this is not as trivial a fix as I had thought; but, I will continue working on it.

Reply:
@emilk 

I cannot figure out how to perform this lint without running `cargo doc` or something similar.  But, if I replace the above line of code with `#![deny(rustdoc::missing_crate_level_docs)] ` and remove the crate documentation comments, this will indeed fail:
`cargo +nightly doc --no-deps --package egui_demo_lib `

I have not yet been able to get this to fail only when including the `--release` cargo doc flag.

How would you like me to proceed?

- Do you want me to push this change to make cargo doc fail any time crate level documentation is missing?  
- I can just remove the current missing_crate_level_docs line so it does not become an issue later.
- Something else; if so, please advise.

Reply:
Oh, so there is no way to get that warning without nightly. Yeah, let's just remove it.